modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/list/List.js

```diff
@@ -59,7 +60,11 @@ const Title = ({item, title, titleRenderer}) => {
 	}
 
 	if (title) {
-		return <ClayList.ItemTitle>{item[title]}</ClayList.ItemTitle>;
+		return (
```

Comment:
We have `link` & `label` properties for the List view, probably to allow displaying title as a link, in the same way we are doing in [FDSViews](https://github.com/liferay-frontend/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSViews.tsx#L290).

Maybe we need to define a custom "opinionated" TitleRenderer just in case we have those `link` & `label` properties mapped. cc/ @markocikos @dsanz

Reply:
Could you please clarify what `link` and `label` props do you mean? `List` component in FDS, in this file, only uses `{header, items, schema}`, and list visualization mode in DSM only has a set of section mappings to fields.

Reply:
Those the user could map in the DSM but don't have a match in the component

![Screenshot from 2024-03-06 16-14-35](https://github.com/liferay-frontend/liferay-portal/assets/132653342/b981baf9-03a7-48f0-9f4d-1f4295cad7af)

Reply:
So, basically, we are providing them in the `schema`

Reply:
Hmmm, I didn't notice before, but these list sections are wrong. In the [implementation](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/list/List.js#L81) we have:
- description
- image
- sticker
- symbol
- title

This means:
1. We are missing `Image` in DSM.
2. We are missing `Sticker` in DSM.
3. `Label` in DSM makes no sense as a List Section. It is not a building block tied to a field. It is the value in FDS that we place in dropdown for views. For views created by DSM, this will always be hardcoded to `List`, unless in future we want to support multiple visualizations modes of the same type.

This would be a product change, pinging @dsanz @herrardo

Reply:
@markocikos  @dsanz  Should this be a blocker? The goal of the Story is to allow changing visualization mode. 
We could define better requirements for each mode in a follow up task.

Reply:
yeah, this is for sure out of the scope of this task

Reply:
I guess we have not correctly mapped the Fields to be exposed to the admin user when creating a View list, with the fields we need on the FDS List side. We can review it, would you like me to create a task to do so in next refinement?

Reply:
I have to review my notification settings, i was not getting this mentions in the email, i discovered this just by chance

Reply:
@herrardo yes, please create a task. We need to do it as a part of the list and cards epic.

As for notifications, I highly recommend `Participating` tab on Github. There is a lot of noise on all Liferay forks, this filters out the relevant bits.

![image](https://github.com/liferay-frontend/liferay-portal/assets/5435511/18659e8e-48f6-4b63-93e6-f48a056edc5f)